src/server/config/env/runTime.js

```diff
@@ -252,5 +252,5 @@ const runTime = [
   },
 ];
 runTime.forEach(preprocessEnvVar);
-export { ip };
```

Comment:
`ip` is actually a `string` but we return a function, should we return the function call?